great_expectations/analytics/base_event.py

```diff
@@ -85,8 +86,11 @@ def properties(self) -> dict:
             "service": "gx-core",
             "user_agent_str": self.user_agent_str,
             "mode": self.mode,
+            # https://posthog.com/docs/libraries/python#person-profiles-and-properties
+            "$process_person_profile": not get_config().anonymize_events,
         }
-        if self.user_id is not None:
+
+        if self.user_id is not None and not get_config().anonymize_events:
```

Comment:
@billdirks

Reply:
it's not entirely the same as before. We don't need this based on the posthoc doc. I'll remove it.

Reply:
Thanks!